bricks/test_optimizer/__brick__/test/.test_optimizer.dart

```diff
@@ -6,11 +6,11 @@
 import 'package:flutter_test/flutter_test.dart';
 {{/isFlutter}}{{^isFlutter}}import 'package:test/test.dart';{{/isFlutter}}
 
-{{#tests}}import '{{{.}}}' as {{#snakeCase}}{{{.}}}{{/snakeCase}};
+{{#tests}}import '{{{path}}}' as {{identifier}};
 {{/tests}}
 void main() {
 {{#isFlutter}}  goldenFileComparator = _TestOptimizationAwareGoldenFileComparator();{{/isFlutter}}
-{{#tests}}  group('{{#snakeCase}}{{{.}}}{{/snakeCase}}', () { {{#snakeCase}}{{{.}}}{{/snakeCase}}.main(); });
+{{#tests}}  group('{{path.snakeCase()}}', () { {{identifier}}.main(); });
```

Comment:
@wolfenrain wdyt of removing the .snakeCase here?

So the group has the file name:
```dart
group('app/view/app_test.dart', () {})
// Instead of 
group('app_view_app_test_dart', () {})
```

Reply:
That would fail if the path contains a `'` which is a technically possible path. I think that is why we originally did the snakeCase as that removes all of those issues

Reply:
Awesome, I'll keep it as is and update the README.